src/components/page-types/formPage/tripNotifyMe.js

```diff
@@ -54,31 +52,31 @@ const TripNotifyMe = (props) => {
       <Grid xs={12}>
         <GridCell xs={12} md={6} lg={12}>
           <SbLink
-            classes="su-group su-inline-block su-font-light su-rs-mb-6 su-no-underline su-transition-colors"
+            classes="su-group su-inline-block su-font-semibold su-basefont-23 su-rs-mb-6 su-no-underline su-transition-colors su-text-digital-red-xlight hocus:su-text-white hocus:su-underline hocus:su-underline-offset-[3px]"
             link={{ url: tripURL }}
           >
             <HeroIcon
               iconType="arrow-left"
-              className="su-inline-block su-text-digital-red-light group-hocus:su-text-cardinal-red"
+              className="su-inline-block su-text-digital-red-xlight group-hocus:su-text-white"
               isAnimate
             />
             Back to {tripTitle}
           </SbLink>
           <Helmet titleTemplate={title} title={title} />
           <Heading
             level={1}
+            size={6}
             align="left"
             font="serif"
             id="page-title"
             className="su-drop-shadow"
           >
             {title}
           </Heading>
-          {hasRichText(body) && (
-            <RichTextRenderer
-              wysiwyg={body}
-              className="su-card-paragraph su-rs-mb-5 children:su-leading-snug children:!su-mb-06em children:last:!su-mb-0"
-            />
+          {body && (
+            <p className="su-intro-text children:su-leading-snug children:!su-mb-06em children:last:!su-mb-0">
```

Comment:
```suggestion
            <p className="su-intro-text su-rs-mb-5">
```
Since this is not a WYSIWYG wrapper anymore, there's no children within the paragraph so removing the classes that we no longer need.

Also, tweaking spacing below the paragraph with spacer 5 😄 
![Screen Shot 2022-07-08 at 11 48 30 AM](https://user-images.githubusercontent.com/42749717/178054122-de37aa0a-669c-44fd-af46-2895698b7cbc.png)